dts/bindings/sensor/sensirion,scd30.yaml

```diff
@@ -9,3 +9,11 @@ include: i2c-device.yaml
 properties:
     label:
       required: true
+    sample-period:
+      type: int
+      required: false
+      default: 30
+      description: |
+        The initial sample period in seconds. The sensor will be
+        configured with this value on start up. This values must
+        be in the rang [2; 1800]. This value be changed at run time.
```

Comment:
default needs to be justified, see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#dt-bindings-default
scd30 seems to use `2` as default

Reply:
The default of 2 seconds is only a valid when the sensor is first powered on of from the factory. When the user of the sensor
changes this values it is stored non volatile memory. This is what the [datasheet](https://www.sensirion.com/fileadmin/user_upload/customers/sensirion/Dokumente/9.5_CO2/Sensirion_CO2_Sensors_SCD30_Interface_Description.pdf) says in chapter 1.4.3.:

> Sets the interval used by the SCD30 sensor to measure in continuous measurement mode (see chapter 1.4.1). Initial value is  2 s. The chosen measurement interval is saved in non-volatile memory and thus is not reset to its initial value after power up.

I think having a default here is valuable, but maybe changing it to 2 is a better value since it comes from the factory this way.